### ISSUE TITLE
Upgrade React version

### ISSUE THREAD

**Issue by Manuelandro** [omgovich/react-colorful#214]
Can you support react the latest version? I got conflict when i try to upgrade react in my project. The upcoming 19 release would be awesome. Thanks

**Comment by kayes360** [comment]
same problem

**Comment by elijaholmos** [comment]
@omgovich pinging for visibility

**Comment by omgovich** [comment]
Hey guys! @followdarko is working on the fix.
Could you pls describe what type of "conflict" you encountered? That would help us to be sure we fix it in #221 
Is it TS types incompatibility or some runtime error?

**Comment by followdarko** [comment]
Indeed! I tested on `create-next-app` with `react@19` / `node@22` / `npm@10` everything works well.

> I got conflict when i try to upgrade react in my project

It's probably not directly related to `react-colorful`. Also `peerDependencies` is already set to `"react": ">=16.8.0"` so it should work correctly.

@Manuelandro @kayes360 @elijaholmos 
Could you share your node js and package manager name and version please?
